backend/src/main/java/com/bakdata/conquery/models/preproc/PreprocessingJob.java

```diff
@@ -33,7 +33,8 @@ public File getPreprocessedFile() {
 			return preprocessedOut.resolve(descriptor.getName() + EXTENSION_PREPROCESSED).toFile();
 		}
 
-		return preprocessedOut.resolve(descriptor.getName() + "." + tag.get() + EXTENSION_PREPROCESSED).toFile();
+		// With tag, name is overwritten to tag
+		return preprocessedOut.resolve(descriptor.getTable() + "." + tag.get() + EXTENSION_PREPROCESSED).toFile();
```

Comment:
name ist einfach eine Jahres-Spanne?

Reply:
name wird in PreprocessorCommand auf das tag gesetzt, also eine Jahresspanne. Das macht die Namen der Imports sauberer.

Reply:
Zeile 3001